src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,296 @@
+
+-   Change password
+-   Success & welcome
+
+<React.Fragment>
+    <ImageGrid
+        images={[ChangePassword, ChangePasswordWelcome]}
+        caption={['1: Change password', '2: Success & welcome']}
+    />
+    <Divider />
+</React.Fragment>
+
+# Behavior
+
+## Email Communications
+
+<!-- This might deserve its own pattern. We already have something in the backlog to design for an email template. -->
+
+Email communications are frequently used throughout the user authentication workflow since they typically involve some security changes. You should never give the users option to unsubscribe from account-related email communications.
+
+<ImageGrid
+    images={[EmailCommunicationInvitation, EmailCommunicationVerification]}
+    caption={['Two examples of email communications', '1: Invitation to join', '2: Verifying email address']}
+/>
+
+## Success & Welcome Screen
+
+<!-- Welcome screen might deserve to be its own pattern (or at least merge into something like "onboarding"). -->
+<!-- TODO: add more images here, if studio blue can release more. -->
+
+The purpose of a welcome screen is to assure the user that something has been indeed achieved, and thus can be more delightful. You are encouraged to use animations and [graphics](/style/illustrations).
+
+## Terminated Workflows & Expired Sessions
+
+During the registration, if a user exits the application before they ever create a password for their account, the whole session is treated as expired, and the users should start over again.
+
+If an invitation link or verification code has expired, the users should be able to request for another invitation link or verification code.
+
+## Security Concerns
+
+### Error Messages
+
+When a user encounters an error, whether it is a password mismatch or the server is temporarily down, they should be notified with an appropriate error message. However, be very careful when presenting error messages to the users. You should only show what is necessary to recover from the problem, such as "incorrect email or password" and "server unavailable".
+
+<ImageGrid
+    images={[ErrorStates]}
+    caption={
+        'For security, you should not present more information than is necessary. For invalid credentials, you should highlight both fields and say "incorrect email or password."'
+    }
+    regularWidth
+/>
+
+### Credential Leaks
+
+Unlike a phone application, a shared HMI application has more threats in credential leaks. These projects should grant their user neither the ability to toggle their password's visibility nor should the application remember users' credentials (i.e., "Remember Me").
+
+### Two-Factor Authentication
+
+The examples on this page used email as the login credentials. However, if your project seeks for additional securities, consider using two-factor authentication.
+
+<Divider />
+
+# Developer
```

Comment:
Are we limiting these components to just the workflow packages, or should we also include some of the basics if they want to roll their own:
Input, Card, etc.?

Reply:
I am inclined to picture them as "we recommend using these pxb packages"

Those inputs & cards should already be mentioned in form pattern & overlay pattern, so I don't think it is worth repeating them here.

Reply:
I think that's fine. I would just match the same verbiage we use in the other developer sections "You should use the following packages to implement this pattern:"

And then maybe link out to the Forms and Overlays patterns again. Something like:

"Additionally, you should familiarize yourself with the components used for the Forms and Overlays patterns".